src/Core/src/Platform/iOS/UIApplicationExtensions.cs

```diff
@@ -72,6 +72,11 @@ internal static UIEdgeInsets GetSafeAreaInsetsForWindow(this UIApplication appli
 			}
 #pragma warning restore CA1416
 
+			if (!OperatingSystem.IsIOSVersionAtLeast(13))
+				return null;
+			else if (windowScene.Delegate is MauiUISceneDelegate sd)
```

Comment:
I think you can cast to `IUIWindowSceneDelegate` instead as that is where the window property comes from.

Reply:
Nice! Updated all the code to use `IUIWindowSceneDelegate.GetWindow()` instead